npm/src/json.ts

```diff
@@ -1,4 +1,4 @@
-import * as Deimos8 from '../../registry/chains/penumbra-testnet-deimos-8.json';
+import * as Deimos8 from '../../registry/chains/penumbra-testnet-deimos-8-x6de97e39.json';
```

Comment:
It looks like the `penumbra-testnet-deimos-8.json` has been removed from the inputs. To pass the test, these need to be updated.

Reply:
Related to https://github.com/prax-wallet/registry/pull/73, last time L and I were debugging. IIRC we were surprised to see that the relevant codepaths were dropping the `-x6de97e39` and trying to import the suffix-less JSON file, which is _not_ what I wanted. I suspect this is still an issue, but we can follow up and file if testing surfaces it again.

Reply:
Discussed offline: made an update to the chain id swapping logic to only do so if the exact match fails